packages/react-instantsearch-dom/src/components/RangeInput.js

```diff
@@ -97,29 +97,32 @@ export class RawRangeInput extends Component {
         className={classNames(cx('', !canRefine && '-noRefinement'), className)}
       >
         <form className={cx('form')} onSubmit={this.onSubmit}>
-          <input
-            className={cx('input', 'input--min')}
-            type="number"
-            min={min}
-            max={max}
-            value={from}
-            step={step}
-            placeholder={min}
-            disabled={!canRefine}
-            onChange={e => this.setState({ from: e.currentTarget.value })}
-          />
-          <span className={cx('separator')}>{translate('separator')}</span>
-          <input
-            className={cx('input', 'input--max')}
-            type="number"
-            min={min}
-            max={max}
-            value={to}
-            step={step}
-            placeholder={max}
-            disabled={!canRefine}
-            onChange={e => this.setState({ to: e.currentTarget.value })}
-          />
+          <label className={cx('label')}>
```

Comment:
This is incorrect, the `label` should not wrap both inputs. We should have two labels: one for each input. I don't understand why we put a `label` around the input in the first place,  we don't provide that possibility to add a text.

> https://instantsearch-css.netlify.com/widgets/range-input/

Reply:
Do you think we should rather remove the label from the spec?

Reply:
If it does not bring any value, yes.

Reply:
Right – closing this PR then.